src/main/java/org/valdroz/vscript/BaseNode.java

```diff
@@ -353,9 +353,40 @@ public Variant execute(VariantContainer variantContainer) {
                 if ((result == null || result.isNull()) && (valueSubstitution != null)) {
                     result = valueSubstitution.execute(variantContainer);
                 }
+
                 result = Variant.sanitize(result);
             }
             break;
+            case NT_MF_IF: {
```

Comment:
`case NT_MF_IF:` could be modeled after `case NT_MF_TO_ARRAY:`

Reply:
`NT_MF_IF` modeled after `NT_MF_TO_ARRAY`